documents/HgbInternshipReportEN/main.tex

```diff
@@ -0,0 +1,129 @@
+\begin{document}
+%%%-----------------------------------------------------------------------------
+
+%%%-----------------------------------------------------------------------------
+% Title page entries
+%%%-----------------------------------------------------------------------------
+
+\title{Final report on the internship at Mogulovich International}
+%\subtitle{Subtitle only if absolutely necessary}
+\author{Alex A.\ Wiseguy}
+
+\programtype{Bachelor's Degree Program}
+\programname{Universal Computing}
+\institution{University of Applied Sciences Upper Austria}
+
+\placeofstudy{Hagenberg}
+\dateofsubmission{2025}{07}{01} % {JJJJ}{MM}{TT}
+
+% List of advisors, up to 4 are possible, title in [] is optional
+\advisor{Pjotr I.~Czar, M.A.}
+%\advisor[Team Lead]{Bettina B. Ackend}
+
+\companyName{%
+   Mogulovich International Media GmbH\\
+   Online Division\\
+   Hubertusgasse 3a, 1020 Wien
+}
+\companyUrl{www.mogul.at}
+
+\license{cc}      % Publish under Creative Commons License (recommended)
+%\license{strict} % Restrictive license, "all rights reserved"
+
+%%%-----------------------------------------------------------------------------
+\frontmatter                                   % Front part (roman page numbers)
+%%%-----------------------------------------------------------------------------
+
+\maketitle
+\tableofcontents
+
+%%%-----------------------------------------------------------------------------
+
+\chapter{Abstract}
+
+Scope of the abstract: approx.\ 200 words.
+
+General content of the report: This report describes the course of the
+internship, the tasks and projects carried out and the experiences gained. The
+own activities (projects) are of course the focus and form the main part of the
+report. If many small projects were worked on, some of them should be described
+in more detail as examples. In addition to the actual work, the following other
+aspects should also be taken into account:
+%
+\begin{itemize}
+	\item Workflows within the company or in projects (graphic representations
+	can be useful here),
+	\item Working and management style, communication within the company,
+	\item External communication (customers, partners),
+	\item Time situation, deadline problems,
+	\item Embedding in the team, social experiences,
+	\item Use of special techniques, methods and tools,
+	\item Important challenges or difficulties,
+	\item Requirements with regard to training during studies (knowledge that
+	can be used well, deficits).
+\end{itemize}
+%
+The following chapter headings are only intended to serve as an orientation for
+the structure of the report; you can of course decide on the specific division
+and wording yourself.
+
+%%%-----------------------------------------------------------------------------
+\mainmatter                                    % Main part (arabic page numbers)
+%%%-----------------------------------------------------------------------------
+
+\chapter{The company}
+
+Scope: 1--2 pages
```

Comment:
"Extent" (or "length") is the more fitting translation here, as scope relates more to the content.